### PR TITLE
Dev/jsequeirag/Color Changes 

### PR DESCRIPTION
**Pull request by DieS06**

### CODE REVIEW

**Review by kendallduranv — CHANGES_REQUESTED** [review]
Envio fotos de lo encontrado en whats app

**Review by mur-tz — APPROVED** [review]

File: src/app/pages/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="container-fluid my-5">
-  <p class="fs-5 pt-5 mb-5">Hello <span class="fw-bold">{{ userName }}</span>, this is an overview of your activity.</p>
+  <p class="fs-5 pt-5 mb-5">Hello_  <span class="fw-bold">{{ userName }}</span>, this is an overview of your activity.</p>
```
Comment (nefi-cenfo) [inline src/app/pages/dashboard/dashboard.component.html:2]: Is this underscored intended?

File: src/app/pages/flights/flights.component.scss [hunk 2]
```diff
@@ -17,14 +16,25 @@ $title-color: #6D3652;
...
+    width: 5%;
+    display: flex;
+    justify-content: center;
+    padding: 1rem;
 }
 
 .logo {
-    width: 2.5vw;
-    height: 5vh;
+    width: 6vh;
```
Comment (nefi-cenfo) [inline src/app/pages/flights/flights.component.scss:32]: Dou you mean 6vw?

File: src/app/pages/food/food.component.scss [hunk 3]
```diff
@@ -17,14 +16,25 @@ $title-color: #6D3652;
...
+    width: 5%;
+    display: flex;
+    justify-content: center;
+    padding: 1rem;
 }
 
 .logo {
-    width: 2.5vw;
-    height: 5vh;
+    width: 6vh;
```
Comment (nefi-cenfo) [inline src/app/pages/food/food.component.scss:32]: Do you mean 6vw?

File: src/assets/layout/_sidebar.scss [hunk 4]
```diff
@@ -1,6 +1,36 @@
...
+
+// i:hover {
+//   color: $light-color-main;
+// }
+
+div .header-wrapper {
+  background-color: $neutro-color-second;
+}
+
+// div .header-wrapper span {
```
Comment (nefi-cenfo) [inline src/assets/layout/_sidebar.scss:28]: do you need this commented code?

File: src/styles.scss [hunk 5]
```diff
@@ -108,3 +116,13 @@ body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
     background: #6D3052;
   }
 }
+
+// //Variables de color
```
Comment (nefi-cenfo) [inline src/styles.scss:120]: Do you need this commented code?

**Review by nefi-cenfo — APPROVED** [review]

File: src/app/pages/dashboard/dashboard.component.html (hunk 1 above)
Comment (DieS06) [reply src/app/pages/dashboard/dashboard.component.html:2]: To make a space between the username and the hello, I tried adding " ", and didnt work.

File: src/app/pages/flights/flights.component.scss (hunk 2 above)
Comment (DieS06) [reply src/app/pages/flights/flights.component.scss:32]: Those were deleted.

File: src/app/pages/food/food.component.scss (hunk 3 above)
Comment (DieS06) [reply src/app/pages/food/food.component.scss:32]: Those were deleted.

**Pull request merged**